components/carousel/index.en-US.md

```diff
@@ -0,0 +1,41 @@
+---
+category: Components
+type: Presentation
+title: Carousel
+---
+
+A carousel component. Scales with its container.
+
+## When to use
+
+- There is a group of content on the same level.
+- There is insufficient content space, it can be used to save space in the form of a resolving door.
```

Comment:
**When** there is insufficient content space, it can be used to save space in the form of a resolving door.